### PR TITLE
Correct shortcut for switching terminals on mac

### PR DESCRIPTION
**Pull request by hatched**

### CODE REVIEW

**Review by kimlisa — APPROVED** [review]
🎉

File: .github/ISSUE_TEMPLATE/testplan.md [hunk 1]
```diff
@@ -710,7 +710,7 @@ With the previous role you created from `Strategy Reason`, change `request_acces
 
 ## Terminal
 - [ ] Verify that top nav has a user menu (Main and Logout)
-- [ ] Verify that switching between tabs works on alt+[1...9]
+- [ ] Verify that switching between tabs works with `alt+[1...9]` (ctrl on mac)
```
Comment (zmb3) [inline .github/ISSUE_TEMPLATE/testplan.md:713]: Given that everyone here is going to be running macs, shouldn't we assume mac by default and say `(alt on linux/windows)`??

**Review by codingllama — APPROVED** [review]
Please address Zac's comment.

File: .github/ISSUE_TEMPLATE/testplan.md (hunk 1 above)
Comment (hatched) [reply .github/ISSUE_TEMPLATE/testplan.md:713]: Yeah that seems reasonable 👍

**Review by zmb3 — APPROVED** [review]

**Pull request merged**